x-pack/plugins/security_solution/public/timelines/components/timeline/query_tab_content/index.tsx

```diff
@@ -285,14 +297,19 @@ export const QueryTabContentComponent: React.FC<Props> = ({
     id: timelineId,
     indexNames: selectedPatterns,
     language: kqlQuery.language,
-    limit: itemsPerPage,
+    limit: unifiedComponentsInTimelineEnabled ? sampleSize : itemsPerPage,
```

Comment:
`sampleSize` is definitely a discover term. When we are able to remove the feature flag, we should keep using `itemsPerPage` or `eventCountPerResponse` or something like that

Reply:
Agreed...I like `eventCountPerResponse`. Let me check and make the changes.

Reply:
Do you still plan to make this? If not, no worries, just wanted to check

Reply:
Forgot to reply here. Let's do it in follow up PR.